public/app/plugins/datasource/testdata/query_ctrl.ts

```diff
@@ -37,9 +36,13 @@ export class TestDataQueryCtrl extends QueryCtrl {
   selectors: typeof e2e.pages.Dashboard.Panels.DataSource.TestData.QueryTab.selectors;
 
   /** @ngInject */
-  constructor($scope: IScope, $injector: any) {
+  constructor($scope: any, $injector: any) {
```

Comment:
nit: I understand why you did this but you should probably write `IScope & {selectors: Record<string, any> }`